.gitignore

```diff
@@ -7,6 +7,7 @@
 
 ### Linux ###
 *~
+node_modules
```

Comment:
Just so you know, you do not need to add `node_modules` here since it already exists on line 46.